src/class-elements/syntax/valid/cls-decl-elements-valid-syntax-module.template

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2019 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-ClassElement
+name: class declaration module
+path: language/statements/class/elements/syntax/valid/module-
+flags: [module]
+features: [class]
+---*/
+
+class C /*{ heritage }*/{
+  /*{ elements }*/
+}
```

Comment:
how is this related to a module? Just the fact it asserts a class inside a module?

It should better to remove this template. There is a template for Script code that can be reused by the runners as modules if necessary. Otherwise this opens a precedent for every syntax we test in this suite.